docs/admin/infrastructure/kubernetes-node-status-max-images.rst

```diff
@@ -0,0 +1,35 @@
+##########################################################
+Kubernetes kubelet nodeStatusMaxImages setting for Nublado
+##########################################################
+
+Setting nodeStatusMaxImages
+===========================
+
+The image prepuller in the :px-app:`nublado` application requires Kubernetes to keep track of a number of images and ensure each of those images are present on every node.  This is required in order to provide a pleasant user experience, because the ``sciplat-lab`` images are large and typically take 3-5 minutes to pull and unpack when they are not already present on a node.
+The default Kubernetes settings can in some circumstances result in the :px-app:`nublado` failing to display images in its spawner menu, as well as the image prepuller running continuously.
```

Comment:
I'd move these above this section to serve as a page description. Every page needs a paragraph of text to set the context before launching into the specific sections.